test/test.js

```diff
@@ -156,9 +157,50 @@ app.post('/api/v3/campaigns/32a381c49a2df99f1d0c6f3c112352b9.json', function (re
     sendFixture(res, 'create_campaign');
 });
 
+//---------------------------------------------------------------------------
+// Transactional requests
+//---------------------------------------------------------------------------
+app.get('/api/v3.1/transactional/smartemail/', function (req, res) {
+    sendFixture(res, 'smart_email_list');
+});
+
+app.get('/api/v3.1/transactional/smartemail/53e872f5-d8d7-46c9-b3d2-e7c7fc650708', function (req, res) {
+    sendFixture(res, 'smart_email_details');
+});
+
+app.post('/api/v3.1/transactional/smartemail/53e872f5-d8d7-46c9-b3d2-e7c7fc650708/send', function (req, res) {
+    sendFixture(res, 'send_smart_email');
+});
+
+app.post('/api/v3.1/transactional/classicEmail/send', function (req, res) {
+    sendFixture(res, 'send_classic_email');
+});
+
+app.get('/api/v3.1/transactional/classicEmail/groups', function (req, res) {
+    sendFixture(res, 'classic_email_group_list');
+});
+
+app.get('/api/v3.1/transactional/statistics', function (req, res) {
+    sendFixture(res, 'statistics');
+});
+
+app.get('/api/v3.1/transactional/messages/', function (req, res) {
```

Comment:
This should be:

    app.get('/api/v3.1/transactional/messages', function (req, res) {